README.md

```diff
@@ -399,30 +399,48 @@ If the transaction was rejected by the node, then `sendTransaction` will throw a
 
 ### Supported Transactions
 
-- TransferTransaction 
+- TransferTransaction: 
```

Comment:
Is adding these colons really necessary? Please keep PRs as focused as possible. It will make it easier to review and merge.